app/src/main/java/com/github/se/polyfit/ui/theme/Color.kt

```diff
@@ -16,3 +16,4 @@ val GradientButtonPink = Color(0xFF4A00E0)
 val GradientButtonPurple = Color(0xFF8E2DE2)
 val GradientButtonGrey = Color(0xFFE0E0E0)
 val DeleteIconRed = Color(0xFFFF7575)
+val BorderPurple = Color(0xFFD8D1F5)
```

Comment:
No action needed for this PR, but we should add a ticket to backlog to let the overview screen use this proper border color as well